### PR TITLE
March 2nd committee meeting

### PR DESCRIPTION
**Pull request by BillWagner**
First commit has proposed agenda.

I'll update after the meeting with the full minutes.

Add any agenda items.

### CODE REVIEW

**Comment by BillWagner** [comment]
@wordshaker @khalidabuhakmeh @jguadagno @clairernovotny @ovebastiansen 

This is ready for review. Any comments to add?

File: meetings/2021-03-02-meeting-minutes.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Meeting minutes: Tuesday March 2, 2021
+
+## Chair report: News from February board meeting
+
+The Membership Committee discussed the outcomes of the February Board Meeting. There have been various discussions around project copyrights and updating the Foundation’s project documentation. Due to this, all project onboarding was stopped until further notice. Claire shared an update with the Committee that the project onboarding process will commence. All Committee members were encouraged to take the survey and share with others. The .NET survey will close in two weeks.
+
+## Create proposal: Membership profiles
+
+Complete review of [Membership Profile pull request](https://github.com/dotnet-foundation/wg-membership/pull/14) and approve current implementation.
```
[the remark this answers is not in the document]
Comment (wordshaker) [reply meetings/2021-03-02-meeting-minutes.md:9]: This terminology comes up a few times in this doc.

**Comment by wordshaker** [comment]
> @wordshaker @khalidabuhakmeh @jguadagno @clairernovotny @ovebastiansen
> 
> This is ready for review. Any comments to add?

Sorry Bill 😂 just a couple of comments

File: meetings/2021-03-02-meeting-minutes.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+## Upcoming board election: Develop schedule
+
+The Membership Committee reviewed the proposed nomination schedule. The Committee also reviewed the Foundation’s Election Policy document. The Committee held a discussion on how the Nomination Committee will handle Board Members running for re-election. They discussed rather they will be automatically added to the election slate or if they will need to be nominated by the Nomination Committee. They also discussed the need for transparency around the election process. 
+
+**Actions**:
+
+- Bill Wagner to reach out to Beth Massi for the RD & MVP Communities email list for the Nominating Committee.
+- Tom Pappas to send the Membership Committee suggested language to use for candidates who were not chosen.
+
```
[the remark this answers is not in the document]
Comment (BillWagner) [reply meetings/2021-03-02-meeting-minutes.md:27]: Right. I'll add that you were creating the timeline (which we approved this morning.)

**Pull request merged**